robot/description/urdf/inertias.xacro

```diff
@@ -34,6 +34,16 @@
             izx="0.0000" izy="0.0000" izz="0.0231"
         />
     </xacro:macro>
-    
+
+    <xacro:macro name="box_inertia" params="mass length width height">
+        <inertia
+            ixx="${mass * (height*height + length*length)/12.0}" 
+                        ixy="0.0" 
+            ixz="0.0" 
+                        iyy="${mass * (width*width + length*length)/12.0}" 
+                        iyz="0.0" 
+            izz="${mass * (width*width + height*height)/12.0}"
```

Comment:
inconsistent spacing, either choose between "space" or "no space" b/w arithmetic operator and operands